### DISCUSSION TITLE
Notification history of user

### DISCUSSION THREAD

**Discussion by darkLord19** [mattermost/community#109] [Moonshot Ideas]
It could include every time user was directly or indirectly mentioned in a conversation

**Comment by esethna** [comment]
Do you mean that you personally could subscribe to mentions of other users?

**Reply by darkLord19** [reply]
No. User can only see their own notification history. Helps to quickly go through conversations in which you were mentioned

**Reply by Willyfrog** [reply]
What would be the difference with the `@` button at the top?

**Reply by darkLord19** [reply]
you mean @ button at the top in channel?

**Reply by Willyfrog** [reply]
yes, that will show you all your mentiones sorted, right? It might not help with the problem you might want to solve though.

**Reply by esethna** [reply]
Yep, sounds like this @ button (Recent mentions):

![image](https://user-images.githubusercontent.com/14333569/123450429-96afa600-d591-11eb-9b34-70271082d6dd.png)

**Reply by darkLord19** [reply]
this is per channel basis right. What I had proposed is global

**Reply by esethna** [reply]
This is actually per team, but it SHOULD be globally in an ideal world

**Comment by chetanyakan** [comment]
An interesting feature here that could be done inside a plugin is to mark a notification as done.
Or to automatically mark the notification as done when posting a message in the channel you were mentioned in.

**Reply by esethna** [reply]
Interesting, can you describe this more? I'm not sure I understand what you mean

**Reply by chetanyakan** [reply]
I mean to show a list of user's pending conversations, including at-mentions and DMs.

And allow mark individual or all conversations as done.

Does that make sense?

**Reply by esethna** [reply]
Yep we have a related ticket here: https://mattermost.atlassian.net/browse/MM-2541
